src/frontend/context/SettingsContext.tsx

```diff
@@ -54,12 +53,30 @@ export const SettingsProvider = ({ children }: React.PropsWithChildren<{}>) => {
     [setState]
   );
 
+  /**
+   * @typedef {SettingsContextType[2]} setExperiment
+   * @param key The key of the experiment to toggle
+   * @param value The value to set the experiment to. If not provided the value will be toggled
+   * @description Sets the value of the experiment to the provided value or toggles it if no value is provided
+   */
```

Comment:
Curious why the JS doc annotation here?

Reply:
I was curious about getting intellisense to pick up the description because i do think it is an odd use case, And there is the option to not set a value, so i just thought it would be useful to have that info while coding...But I don't know how to do that with react and jsdoc because this did not work